packages/test/generatedCode/models/microsoft/graph/dateTimeTimeZone.ts

```diff
@@ -1,78 +1,9 @@
-     * @returns a Record<string, unknown>
-     */
-    public get additionalData() {
-        return this._additionalData;
-    };
-    /**
-     * Sets the additionalData property value. Stores additional data not described in the OpenAPI description found when deserializing. Can be used for serialization as well.
-     * @param value Value to set for the AdditionalData property.
-     */
-    public set additionalData(value: Record<string, unknown>) {
-        this._additionalData = value;
-    };
-    /**
-     * Instantiates a new dateTimeTimeZone and sets the default values.
-     */
-    public constructor() {
-        this._additionalData = {};
-    };
-    /**
-     * Gets the dateTime property value. A single point of time in a combined date and time representation ({date}T{time}; for example, 2017-08-29T04:00:00.0000000).
-     * @returns a string
-     */
-    public get dateTime() {
-        return this._dateTime;
-    };
-    /**
-     * Sets the dateTime property value. A single point of time in a combined date and time representation ({date}T{time}; for example, 2017-08-29T04:00:00.0000000).
-     * @param value Value to set for the dateTime property.
-     */
-    public set dateTime(value: string | undefined) {
-        this._dateTime = value;
-    };
-    /**
-     * The deserialization information for the current model
-     * @returns a Record<string, (node: ParseNode) => void>
-     */
-    public getFieldDeserializers() : Record<string, (node: ParseNode) => void> {
-        return {
-            "dateTime": n => { this.dateTime = n.getStringValue(); },
-            "timeZone": n => { this.timeZone = n.getStringValue(); },
-        };
-    };
-    /**
-     * Serializes information the current object
-     * @param writer Serialization writer to use to serialize this model
-     */
-    public serialize(writer: SerializationWriter) : void {
-        if(!writer) throw new Error("writer cannot be undefined");
-        writer.writeStringValue("dateTime", this.dateTime);
-        writer.writeStringValue("timeZone", this.timeZone);
-        writer.writeAdditionalData(this.additionalData);
-    };
-    /**
-     * Gets the timeZone property value. Represents a time zone, for example, 'Pacific Standard Time'. See below for more possible values.
-     * @returns a string
-     */
-    public get timeZone() {
-        return this._timeZone;
-    };
-    /**
-     * Sets the timeZone property value. Represents a time zone, for example, 'Pacific Standard Time'. See below for more possible values.
-     * @param value Value to set for the timeZone property.
-     */
-    public set timeZone(value: string | undefined) {
-        this._timeZone = value;
-    };
+export interface DateTimeTimeZone{
+    /** Stores additional data not described in the OpenAPI description found when deserializing. Can be used for serialization as well. */
+    additionalData?:Record<string, unknown>;
```

Comment:
this property doesn't need to be re-defined

Reply:
This is happening in the current main generation and not introduced in this PR. Marking it as a separate issue.

Reply:
https://github.com/microsoftgraph/msgraph-sdk-typescript/blob/4dfd1260694a59f2e09cb60fb6a8ee99990bcf2d/src/models/dateTimeTimeZone.ts#L5